@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1712,7 +1719,23 @@ export class TextElementsRenderer {
         renderParams: RenderParams
     ): boolean {
         const poiLabel = labelState.element;
-        const worldPosition = poiLabel.position;
+        const worldPosition = this.m_tmpVector3.copy(poiLabel.position);
+
+        const worldOffsetShift = getPropertyValue(
+            (poiLabel.poiInfo?.technique as PoiTechnique)?.worldOffset,
+            this.m_viewState.env
+        );
+        const offsetDirection = poiLabel.userData?.offset_direction;
```

Comment:
Here you are depending on gathering all feature attributes in userData (gatherFeatureAttributes flag must be turned on).
The usual way we decode attributes needed for rendering is to read them in the decoder and add them to the DecodedTile.
I guess you didn't want to make the decoded tiles bigger by adding a new attribute? But this actually makes them even bigger since we're storing all attributes in userData.